Model/Api/Customer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Omise\Payment\Model\Api;
+
+use Exception;
+use OmiseCustomer;
+
+class Customer extends Object
+{
+    /**
+     * @param  string $id
+     *
+     * @return Omise\Payment\Model\Api\Error|self
+     */
+    public function find($id)
+    {
+        try {
+            $this->refresh(OmiseCustomer::retrieve($id));
+        } catch (Exception $e) {
+            return new Error([
+                'code'    => 'not_found',
+                'message' => $e->getMessage()
+            ]);
+        }
+
+        return $this;
+    }
+
+    /**
+     * @param  array $params
+     *
+     * @return Omise\Payment\Model\Api\Error|self
+     */
+    public function create($params)
+    {
+        try {
+            $this->refresh(OmiseCustomer::create($params));
+        } catch (Exception $e) {
+            return new Error([
+                'code'    => 'bad_request',
+                'message' => $e->getMessage()
+            ]);
+        }
+
+        return $this;
+    }
+
+     /**
+     * @param  array $params
+     *
+     * @return \Omise\Payment\Model|\Api\Error|self
+     */
+    public function update($params)
+    {
+        try {
+            $this->object->update($params);
+            $this->refresh($this->object);
+        } catch (Exception $e) {
+            return new Error([
+                'code'    => 'bad_request',
+                'message' => $e->getMessage()
+            ]);
+        }
+
+        return $this;
+    }
+
+    /**
+     * TODO: Need to refactor a bit
```

Comment:
(note) not necessary but I think it's better to not expose `OmiseObject` from `Omise-PHP` out to `Model`.
Things should be encapsulated inside the `Model\Api` structure so we can handle 3 layouts

```
Controller ---> Model ---> Model\Api (equalvalent to ActiveRecord or Eloquent) --- connect to Omise API using Omise-PHP
```